src/swarm/client/registry/NodeRegistry.d

```diff
@@ -232,7 +232,7 @@ public abstract class NodeRegistry : INodeRegistry
         {
             auto handled =
                 this.assignClientCommand(this.getClientCommandParams(params));
-            assert(handled, "unhandled client command code");
+            enforce(handled, "unhandled client command code");
```

Comment:
This looks more like `verify` as it can fail if `params` is invalid. `enforce` is meant to verify input to the program.

Reply:
The trick is that I suspect that the client command code could arrive from the external input, following dependencies. `assignClientCommand` may not cover all possible client command codes.

Reply:
Wait, I'm wrong, this is done on _client_ side.

Reply:
This should indeed be a `verify`.